project/MimaFilters.scala

```diff
@@ -24,6 +24,26 @@ object MimaFilters extends AutoPlugin {
     // with JDK 11 and run MiMa it'll complain IteratorWrapper isn't forwards compatible with 2.13.0 - but we
     // don't publish the artifact built with JDK 11 anyways
     ProblemFilters.exclude[DirectMissingMethodProblem]("scala.collection.convert.JavaCollectionWrappers#IteratorWrapper.asIterator"),
+
+    // Fixes for scala/bug#12009
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.collection.mutable.ArrayBufferView.this"),
```

Comment:
IIUC it’s OK to add these filters because this method and the following ones are package private, right?

Reply:
yes. I forgot to note that in a comment - I'll try to do that

Reply:
fixed in the other PR's branch - will rebase on that once merged